src/EditorFeatures/CSharpTest/CommentSelection/CSharpToggleLineCommentCommandHandlerTests.cs

```diff
@@ -310,7 +310,7 @@ class C
 {
     void M()
     {
-[|        //string s = '\\';|]
+        [|//string s = '\\';|]
```

Comment:
This has changed the selection by adding extra chars at the beginning.
Please leave the selection unchanged by keeping the caret immediately to the left of "string".